engine/runtime/src/main/scala/org/enso/interpreter/instrument/execution/CommandExecutionEngine.scala

```diff
@@ -1,15 +1,15 @@
 package org.enso.interpreter.instrument.execution
 
-import java.util.concurrent.Executors
-import java.util.logging.Level
-
 import org.enso.interpreter.instrument.InterpreterContext
 import org.enso.interpreter.instrument.command.Command
 import org.enso.interpreter.instrument.execution.Completion.{Done, Interrupted}
 import org.enso.interpreter.runtime.control.ThreadInterruptedException
 import org.enso.polyglot.RuntimeOptions
 import org.enso.text.Sha3_224VersionCalculator
 
+import java.util.concurrent.Executors
+import java.util.logging.Level
```

Comment:
Just an import rearrangements. We shouldn't let IDEs of our developers fight for their favorite order of imports. There shall be a formatting rule. CCing @hubertp